config/definitions.yml

```diff
@@ -25,6 +25,12 @@ sle12_common: &sle12_common
     - :server: "http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update/standard/"
       :packages: ["hwinfo"]
 
+:sles12-sp3:
+  <<: *sle12_common
+  :yast_url: "http://download.suse.de/ibs/Devel:/YaST:/Head/sle12_sp3/"
```

Comment:
NP: one day in the future this will be renamed from :Head to ... something else. So, the tests will most probably break or not work as expected. What is benefit of creating the target now?